launcher/ui/pages/instance/ExternalResourcesPage.ui

```diff
@@ -179,15 +171,15 @@
     <string>Reset Update Metadata</string>
    </property>
    <property name="menuRole">
-    <enum>QAction::NoRole</enum>
+    <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
   <action name="actionVerifyItemDependencies">
    <property name="text">
     <string>Verify Dependencies</string>
    </property>
    <property name="menuRole">
-    <enum>QAction::NoRole</enum>
+    <enum>QAction::MenuRole::NoRole</enum>
```

Comment:
It is weird that the qt designer added the `::MenuRole` in the middle.
Not blocking but I just think it will not be consistent with other places.
And may even impact other stuff that we are not aware of.